### PR TITLE
feature: new role for UWG members

### PR DESCRIPTION
**Pull request by joncarlson**
# Description

Adding a role for UWG members, to support the GES DISC task of assigning UWG members to a review

## Spec

Designs: [link to design if applicable; delete if not]

See Ticket: [link to ticket if applicable; delete if not]

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Navigate to... [continue instructions]

_(For an example of good validation instructions, check out
[Bryan's Bouncy Ball PR](https://github.com/sparkbox/bouncy-ball/pull/56#issue-192153701).)_

---

## Change Log

_(copy/paste-able change log notes. check the box when the change is also in CHANGELOG.md)_

* [Add _____](link to commitid)
* [Fix _____](link to commitid)

### CODE REVIEW

File: src/postgres/7-seed.sql [hunk 1]
```diff
@@ -344,6 +345,13 @@ INSERT INTO privilege VALUES ('QUESTION_DELETE');
 -- RolePrivilege(edprole_id, privilege) Administrator
 INSERT INTO edprole_privilege VALUES ('75605ac9-bf65-4dec-8458-93e018dcca97', 'ADMIN');
 
+-- RolePrivilege(edprole_id, privilege) UWG Member
+INSERT INTO edprole_privilege VALUES ('19ac227b-e96c-46fa-a378-cf82c461b669', 'REQUEST_READ');
+INSERT INTO edprole_privilege VALUES ('19ac227b-e96c-46fa-a378-cf82c461b669', 'REQUEST_REVIEW');
+INSERT INTO edprole_privilege VALUES ('a5b4947a-67d2-434e-9889-59c2fad39676', 'NOTE_NEW');
```
Comment (DavisGauntUAH) [inline src/postgres/7-seed.sql:351]: Please fix the edprole_id.  They should be `19ac227b-e96c-46fa-a378-cf82c461b669`

File: src/postgres/9-updates.sql [hunk 2]
```diff
@@ -112,3 +112,11 @@ INSERT INTO step_edge VALUES ('7843dc6d-f56d-488a-9193-bb7c0dc3696d', 'data_publ
 
 UPDATE step SET data = '{"rollback":"data_publication_request_form_management_review","type": "review","form_id":"19025579-99ca-4344-8610-704dae626343"}' WHERE step_id = 'c81066db-0566-428d-87e8-94169ce5a9b9';
 UPDATE step_edge SET next_step_name = 'data_publication_request_form_management_review' WHERE workflow_id = '7843dc6d-f56d-488a-9193-bb7c0dc3696d' AND step_name = 'data_publication_request_form_review';
+
+--4/23/2024 Adding UWG member as a role
+INSERT INTO edprole VALUES ('19ac227b-e96c-46fa-a378-cf82c461b669', 'uwg_member', 'UWG Member', 'A member of the User Working Group who can review requests in Earthdata Pub.');
+INSERT INTO edprole_privilege VALUES ('19ac227b-e96c-46fa-a378-cf82c461b669', 'REQUEST_READ');
+INSERT INTO edprole_privilege VALUES ('19ac227b-e96c-46fa-a378-cf82c461b669', 'REQUEST_REVIEW');
+INSERT INTO edprole_privilege VALUES ('a5b4947a-67d2-434e-9889-59c2fad39676', 'NOTE_NEW');
```
Comment (DavisGauntUAH) [inline src/postgres/9-updates.sql:120]: Same as in seed.sql

**Review by DavisGauntUAH — CHANGES_REQUESTED** [review]

File: src/postgres/9-updates.sql (hunk 2 above)
Comment (joncarlson) [reply src/postgres/9-updates.sql]: Changes made

**Comment by camposeddie** [comment]
@joncarlson @DavisGauntUAH Have the requested changes of this PR been resolved? Is this PR ready to be validated for merging?

**Comment by joncarlson** [comment]
@camposeddie Yes, I've made the requested changes

**Review by camposeddie — APPROVED** [review]

**Comment by camposeddie** [comment]
Everything looks good with the assumption that https://github.com/eosdis-nasa/earthdata-pub-dashboard/pull/50 or similar follow on tasks adds functionality so that UWG member is added as a contributor when assigned to review a request. Otherwise, the UWG role doesn't have any visibility into requests.

**Pull request merged**